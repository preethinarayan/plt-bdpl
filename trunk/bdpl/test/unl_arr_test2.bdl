struct outer
{
	byte size;
	struct inner
	{
		byte[size] n_bytes;
	} u;
}[*]a;

file "x:/test/file_test.dat" inp_file;
read(inp_file,a);
print(a);
