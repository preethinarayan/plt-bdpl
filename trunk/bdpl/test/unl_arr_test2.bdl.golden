ARRAY of struct:outer
[
	0 => struct:outer
	{
		size => 0xd,
		n_bytes => ARRAY of TYPE_BYTE
		[
			0 => 0x41,
			1 => 0x42,
			2 => 0x43,
			3 => 0x44,
			4 => 0x45,
			5 => 0x46,
			6 => 0x47,
			7 => 0x48,
			8 => 0x49,
			9 => 0x4a,
			10 => 0x4b,
			11 => 0x4c,
			12 => 0x4d,
		]
		,
	}
	,
	1 => struct:outer
	{
		size => 0x4e,
		n_bytes => ARRAY of TYPE_BYTE
		[
			0 => 0x4f,
			1 => 0x50,
			2 => 0x51,
			3 => 0x52,
			4 => 0x53,
			5 => 0x54,
			6 => 0x55,
			7 => 0x56,
			8 => 0x57,
			9 => 0x58,
			10 => 0x59,
			11 => 0x5a,
			12 => 0xd,
			13 => 0xa,
			14 => 0x0,
			15 => 0x0,
			16 => 0x0,
			17 => 0x0,
			18 => 0x0,
			19 => 0x0,
			20 => 0x0,
			21 => 0x0,
			22 => 0x0,
			23 => 0x0,
			24 => 0x0,
			25 => 0x0,
			26 => 0x0,
			27 => 0x0,
			28 => 0x0,
			29 => 0x0,
			30 => 0x0,
			31 => 0x0,
			32 => 0x0,
			33 => 0x0,
			34 => 0x0,
			35 => 0x0,
			36 => 0x0,
			37 => 0x0,
			38 => 0x0,
			39 => 0x0,
			40 => 0x0,
			41 => 0x0,
			42 => 0x0,
			43 => 0x0,
			44 => 0x0,
			45 => 0x0,
			46 => 0x0,
			47 => 0x0,
			48 => 0x0,
			49 => 0x0,
			50 => 0x0,
			51 => 0x0,
			52 => 0x0,
			53 => 0x0,
			54 => 0x0,
			55 => 0x0,
			56 => 0x0,
			57 => 0x0,
			58 => 0x0,
			59 => 0x0,
			60 => 0x0,
			61 => 0x0,
			62 => 0x0,
			63 => 0x0,
			64 => 0x0,
			65 => 0x0,
			66 => 0x0,
			67 => 0x0,
			68 => 0x0,
			69 => 0x0,
			70 => 0x0,
			71 => 0x0,
			72 => 0x0,
			73 => 0x0,
			74 => 0x0,
			75 => 0x0,
			76 => 0x0,
			77 => 0x0,
		]
		,
	}
	,
]
